cmd/goal/README.md

```diff
@@ -3,27 +3,34 @@
 Unless otherwise noted, it is assumed that the working directory
 begins at the top level of the `go-algorand` repo.
 
-It is also assumed that the main README's installation instructions have been followed and `make install` run so that the `goal` executable is available via `${GOPATH}/bin/goal`. 
+It is also assumed that the main README's installation instructions have been followed and `make install` run so that the `goal` executable is available via `${GOPATH}/bin/goal`.
 You can verify this to be the case by comparing the output of `which goal` with the output of `echo ${GOPATH}/bin/goal`.
```

Comment:
It doesn't actually matter if that's where it is, right? What matters is that the one you installed is somewhere in your PATH.  Since I use `asdf` to manage Go installs, for me, `goal` is in: `~/.asdf/shims/goal`